cmd/juju/application/bundle/bundle.go

```diff
@@ -330,14 +330,7 @@ func verifyBundle(data *charm.BundleData, bundleDir string, verifyConstraints fu
 		return err
 	}
 
-	var errs []string
-	// This method cannot be included within data.Verify because
-	// to verify corresponding series and base match we need to be
-	// able to compare them. The charm package, however, treats bases
-	// and series generically and is unable to do this.
-	if err := verifyMixedSeriesBasesMatch(data); err != nil {
-		errs = append(errs, err.Error())
-	}
+	errs := assertNoSeries(data)
```

Comment:
```suggestion
	errs := verifyNoSeries(data)
```

or `ensureNoSeries`.